src/stylesheets/_variables.scss

```diff
@@ -1,18 +1,18 @@
 $color-blue: #00a7e9 !default;
 $color-blue-light: #24bcf3 !default;
-$color-blue-lighter: #46C4F2 !default;
+$color-blue-lighter: #46c4f2 !default;
 $color-blue-lightest: #d8f3fd !default;
 $color-red: #ef5b5b !default;
 $color-green: #23aa1d !default;
 $color-orange: #fbb045 !default;
-$color-black: #000000 !default;
-$color-grey-darker: #666666 !default;
-$color-grey-dark: #999999 !default;
+$color-black: #000 !default;
+$color-grey-darker: #666 !default;
+$color-grey-dark: #999 !default;
```

Comment:
I prefer 6 digit colors to keep it consistent with all the other colors that are 6 digits.

Reply:
I prefer the shorter notation. I'll wait for @ImreBognarUltimaker to voice his opinion before changing this.

Reply:
I like the 6 digit better, in my opinion this is the kind of shorthand that has more overhead than writing out the full code every time.

Reply:
I'll have to just find solace in the fact that @ianpaschal prefers `rgb()` notation anyway…